src/main/java/org/motechproject/newebodac/web/CampaignMessageController.java

```diff
@@ -0,0 +1,58 @@
+package org.motechproject.newebodac.web;
+
+import java.util.List;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.motechproject.newebodac.dto.CampaignMessageDto;
+import org.motechproject.newebodac.service.CampaignMessageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+public class CampaignMessageController extends BaseController {
+
+  @Autowired
+  private CampaignMessageService campaignMessageService;
+
+  /**
+   * Function requests all campaign messages from the database.
+   * @return All campaign messages from the database.
+   */
+  @RequestMapping(value = "/campaignMessage", method = RequestMethod.GET)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public List<CampaignMessageDto> getAll() {
+
+    return campaignMessageService.getAll();
+  }
+
+  @RequestMapping(value = "/campaignMessage/{id}", method = RequestMethod.GET)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public CampaignMessageDto findById(
+      @PathVariable("id") UUID id) {
```

Comment:
why is this in a new line? change this in all of the controllers

Reply:
@kdondziak I meant why the parameter is in a new line, it should be in one line
`public CampaignMessageDto findById(@PathVariable("id") UUID id) {`